packages/block-library/src/social-link/index.js

```diff
@@ -7,40 +7,21 @@ import { __ } from '@wordpress/i18n';
  * Internal dependencies
  */
 import edit from './edit';
-import socialList from './social-list';
+import metadata from './block.json';
+import variations from './variations';
 
-const commonAttributes = {
-	category: 'widgets',
+const { name } = metadata;
+
+export { metadata, name };
+
+export const settings = {
+	title: __( 'Social Icon' ),
 	parent: [ 'core/social-links' ],
 	supports: {
 		reusable: false,
 		html: false,
 	},
 	edit,
+	description: __( 'Create a link to the wider Web' ),
```

Comment:
This description could be improved. Reading it in isolation doesn't hint what the block does.

Reply:
Yes, same. My initial diff had a comment about it. Ideas?

Reply:
Create an icon link to external sites.

Reply:
I'll try _Display an icon linking to a social media profile or website_